tests/helpers.py

```diff
@@ -406,21 +414,36 @@ def delete_cephblockpool():
         kind=constants.CEPHBLOCKPOOL,
         namespace=defaults.ROOK_CLUSTER_NAMESPACE
     )
-    pool_list = get_cephblockpool_name()
-    for item in pool_list:
-        logger.info(f"Deleting CephBlockPool with name {item}")
-        assert POOL.delete(resource_name=item)
+    logger.info(f"Deleting CephBlockPool with name {cbp_name}")
+    assert POOL.delete(resource_name=cbp_name)
     return True
 
 
-def create_cephfilesystem():
+def create_cephfilesystem(override_fs=False):
```

Comment:
This is risky,
If this function should create a FS, than that's the only thing it should do.
one more thing, you are not keeping the "old" FS before deletion so how do you plan to restore it?

Reply:
will make override_fs as True and for restore will delete old pods and recreate with the same name will it be good?

Reply:
why not using the existing FS?

Reply:
if someone needs to create new fs he/she can call create_cephfilesystem() otherwise if not no need to call in their script so, that time it will use existing one